IBPSA/Resources/src/fluid/geothermal/visualize_borehole_clusters.py

```diff
@@ -0,0 +1,40 @@
+from buildingspy.io.outputfile import Reader
```

Comment:
It's not clear why this file is part of the PR. It does not generate a figure (or at least it does not write it to disk for display in the `info` section). Should this file be removed, or if not, can you add a comment in the file that explains its usage?